src/content/docs/it/tutorial/1-setup/2.mdx

```diff
@@ -0,0 +1,147 @@
+
+3. Assicurati che il terminale sia visibile e che puoi vedere il promt dei comandi, per esempio:
+
+    ```sh
+    user@machine:~/tutorial$
+    ```
+
+    :::tip[Scorciatoia da tastiera]
+    Per alternare la visibilità del terminal, usa <kbd>Ctrl + J</kbd> (macOS: <kbd>Cmd ⌘ + J</kbd>).
+    :::
+
+Ora puoi utilizzare il terminale circoscritto in questa finestra, invece del terminale del tuo computer, per il resto di questo tutorial.
+
+
+
+## Esegui Astro in modalità sviluppo (dev)
+
+In modo di avere una preview dei file del tuo progetto _come un sito_ mentre lavori, avrai bisogno che Astro sia eseguito in modalità sviluppo (dev).
+
+### Avvia il server dev
+
+1. Esegui il comando per avviare il server Astro dev scrivendo nel terminale di VS Code:
+
+    <PackageManagerTabs>
+      <Fragment slot="npm">
+      ```shell
+      npm run dev
+      ```
+      </Fragment>
+      <Fragment slot="pnpm">
+      ```shell
+      pnpm run dev
+      ```
+      </Fragment>
+      <Fragment slot="yarn">
+      ```shell
+      yarn run dev
+      ```
+      </Fragment>
+    </PackageManagerTabs>
+
+    Ora dovresti avere conferma sul terminale che Astro è in esecuzione in modalità dev. 🚀
+
+## Visualizza un'anteprima del tuo sito
+
+I file del tuo progetto contengono tutto il codice necessario per mostrare un sito Astro, ma il browser è responsabile per mostrare il tuo codice come pagine web.
+
+1. Clicca sul link `localhost` sulla finestra del tuo terminale per visualizzare un'anteprima del tuo nuovo sito Astro!
+
+    (Astro usa `http://localhost:4322` di default se la porta `4321` è disponibile.)
+
+    Ecco come il sito iniziale di Astro "Empty Project" dovrebbe sembrare nell'anteprima del browser:
+
+    ![A blank white page with the word Astro at the top.](/tutorial/minimal.png)
+
+:::tip[Utilizzare il server dev Astro]
+
+Mentre il server Astro è in esecuzione in modalità dev, NON potrai eseguire dei comandi nella tua finestra del terminale. Invece, questo riquadro vi darà feedback sull'anteprima del vostro sito.
+
+Puoi fermare il server dev in qualunque momento e ritornare al promt dei comandi inserendo <kbd>Ctrl + C</kbd> nel terminale.
+
+Alcune volte il server dev si fermerà da solo mentre stai lavorando. Se l'anteprima in tempo reale smette di funzionare, ritorna al terminale e riavvia il server dev inserendo `npm run dev`.
+:::
+
+<Box icon="check-list">
+## Checklist
+
+<Checklist>
+- [ ] So creare un nuovo progetto Astro
+- [ ] So avviare il server dev Astro.
+</Checklist>
+</Box>
+
+### Risorse
+
+- <p>[Getting Started with Visual Studio Code](https://code.visualstudio.com/docs/introvideos/basics) <Badge>external</Badge> — un video tutorial per installare, impostare e lavorare con VS Code</p>
```

Comment:
```suggestion
- <p>[Getting Started with Visual Studio Code](https://code.visualstudio.com/docs/introvideos/basics) <Badge>esterno</Badge> — un video tutorial per installare, impostare e lavorare con VS Code</p>
```

Reply:
didn't see this one, thanks for noticing